pkg/services/sqlstore/apikey.go

```diff
@@ -44,7 +43,7 @@ func AddApiKey(cmd *m.AddApiKeyCommand) error {
 			v := updated.Add(time.Second * time.Duration(cmd.SecondsToLive)).Unix()
 			expires = &v
 		} else if cmd.SecondsToLive < 0 {
-			return fmt.Errorf("Negative value for SecondsToLive")
+			return m.ErrInvalidApiKeyExpiration
```

Comment:
Should be `models.ErrInvalidApiKeyExpiration` so the import at the top shouldn't be aliased. Think we decided that, but can't find any documentation about it.

Reply:
ok, but the alias is used also in other statements in the same file
should I change all of them and remove the alias completely?

Reply:
Yes please remove `m`-alias in all your changed files